caikit/runtime/server_base.py

```diff
@@ -76,6 +76,13 @@ def __init__(self, base_port: int, tls_config_override: Optional[aconfig.Config]
 
         self.training_service = training_service
 
+        # create runtime info service
+        self.runtime_info_service: Optional[
```

Comment:
This is currently failing linting with error [`too-many-instance-attributes`](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-instance-attributes.html), thoughts? Ignore or fix?